packages/angular/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@tanstack-query-firebase/angular",
+  "version": "0.0.1",
+  "description": "Tanstack Query bindings for Firebase and Angular",
+  "type": "module",
+  "scripts": {
+    "copyPackageJson": "node copyPackageJson.js",
+    "build": "tsup && npm run copyPackageJson",
+    "watch": "tsup --watch",
+    "test": "vitest --dom --coverage"
+  },
+  "typings": "./index.d.ts",
+  "module": "./index.js",
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "import": "./index.js",
+      "default": "./data-connect/index.js"
+    },
+    "./data-connect": {
+      "import": "./data-connect/index.js",
+      "types": "./data-connect/index.d.ts",
+      "default": "./data-connect/index.js"
+    }
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@dataconnect/default-connector": "file:dataconnect-sdk/js/default-connector",
```

Comment:
```suggestion
    "@dataconnect/default-connector": "workspace:*",
```

Since the [dc-js-sdk](https://github.com/maneesht/tanstack-query-firebase/tree/main/dataconnect-sdk/js/default-connector) has moved back to workspaces via [714f3b2](https://github.com/invertase/tanstack-query-firebase/pull/166/commits/714f3b2c0c63b227176a331482f50d95ef1a1f4d), installation of packages using the workspace results in the error below. the change of reference to workspace would be a fix for this.
<img width="661" alt="image" src="https://github.com/user-attachments/assets/9f336727-1e8e-4ef3-a4d7-5202468e0078" />

Reply:
Done. Thanks for checking on this. I wasn't able to repro initially, but all it took was removing all of my `node_modules`. Should be fixed now.